.github/workflows/build-test.yaml

```diff
@@ -2063,15 +2283,19 @@ jobs:
           cp ./bin/kots /usr/local/bin/kubectl-kots
           sudo apt-get update -y && sudo apt-get install jq -y
       - uses: ./.github/actions/kots-e2e
+        id: kots-e2e
         with:
           test-focus: 'Target KOTS Version'
           kots-namespace: 'target-kots-version'
-          k8s-version: '${{ matrix.k8s_version }}'
-          k8s-arguments: ${{ startsWith(matrix.k8s_version, 'v1.22') && '--egress-selector-mode disabled' || '' }}
+          k8s-distribution: ${{ matrix.cluster.distribution }}
+          k8s-version: ${{ matrix.cluster.version }}
           testim-access-token: '${{ secrets.TESTIM_ACCESS_TOKEN }}'
           testim-branch: ${{ github.head_ref == 'main' && 'master' || github.head_ref }}
           aws-access-key-id: '${{ secrets.E2E_SUPPORT_BUNDLE_AWS_ACCESS_KEY_ID }}'
           aws-secret-access-key: '${{ secrets.E2E_SUPPORT_BUNDLE_AWS_SECRET_ACCESS_KEY }}'
+          replicated-api-token: '${{ secrets.C11Y_MATRIX_TOKEN }}'
+          k8s-cluster-skip-teardown: true
```

Comment:
why?

Reply:
the min/target/range kots version tests all have an additional cli step after that do some validation, so we need to keep the cluster around for that and then remove it using the `cluster-id` output.  we could re-work these tests so that this is all wrapped up in the testim suite, but that seemed out of scope